README.md

```diff
@@ -36,6 +36,45 @@ Additionally, the Git repository **must** also have the [necessary files for set
 - `binder/postBuild`
 - `binder/environment.yml`
 
+### Language agnostic
+
+Create `binder/environment.yml` using `conda`.
+
+```bash
+conda env -n environment-name export > binder/environment.yml
+```
+
+The `binder/environment.txt` should look like [`binder-examples/language-agnostic/environment.yml`](binder-examples/language-agnostic/environment.yml).
+
+It is strongly recommended to pin the version of the dependencies.
+
+### Python
+
+Create `binder/requirements.txt` using `pip`.
+
+```bash
+python3 -m pip freeze > binder/requirements.txt
+```
+
+The `binder/requirements.txt` should look like [`binder-examples/language-agnostic/requirements.yml`](binder-examples/language-agnostic/requirements.yml).
+
+It is strongly recommended to pin the version of the dependencies.
+
+### R
+
+`install.packages()` should **not** be called from the `.qmd` or `.rmd`!
```

Comment:
This could be slightly confusing, as the updated template does mention `install.packages()` (not call though.)

Perhaps:

````md
`install.packages()` or similar commands for installing R packages (e.g. `pak::pkg_install()`, `devtools::install_github()`) should **not** be called from `qmd` or `rmd`!

If they must be there to illustrate the installation process, set the code block to `eval: false`. See `template.qmd` for an example.

````